sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0.1 - Coming Soon
+
+- Fixed an issue that occurs when using a user-assigned managed identity with `ManagedIdentityCredential` ([PR #6134](https://github.com/Azure/azure-sdk-for-js/pull/6134))
```

Comment:
"Fixed an issue that occurs when something is used". Great, but what is the issue? :)

Can we pick things up from your PR description?

```suggestion
- Fixed an issue where an authorization error occurs due to wrong access token being returned by the MSI endpoint when using a user-assigned managed identity with `ManagedIdentityCredential` ([PR #6134](https://github.com/Azure/azure-sdk-for-js/pull/6134))
```

Reply:
Thanks!